main.go

```diff
@@ -52,5 +54,49 @@ func getConfig(jsonFile string) (config *JSONConfigData) {
 		panic(err)
 	}
 	json.Unmarshal([]byte(J), &config)
-	return config
+	return
+}
+
+func getUsers(jsonFile string) (users *UsersData) {
+	users = &UsersData{}
+	J, err := ioutil.ReadFile(jsonFile)
+	if err != nil {
+		panic(err)
+	}
+	json.Unmarshal([]byte(J), &users)
+	return
+}
+
+func selectReviewers(prOwner string, users UsersData) (rev1, rev2 string) {
+	counter:= 12 //positions availables in the array of users
```

Comment:
good point!. Other than that, in the next version I am going to get the users directly from github, because right now if one of the change his/her name, our tool does not get updated.